ert_storage/endpoints/compute/misfits.py

```diff
@@ -0,0 +1,88 @@
+import numpy as np
+import pandas as pd
+from uuid import UUID
+from typing import Any, Mapping, Optional, List
+import sqlalchemy as sa
+from fastapi import (
+    APIRouter,
+    Depends,
+)
+from ert_storage.database import Session, get_db
+from ert_storage import database_schema as ds, json_schema as js
+
+router = APIRouter(tags=["misfits"])
+
+
+def _calculate_misfit(obs_value, response_value, obs_std, x_value):
+    difference = response_value - obs_value
+    misfit = (difference / obs_std) ** 2
+    sign = difference > 0
+    return {
+        "value": misfit.tolist(),
+        "sign": sign.tolist(),
+        "obs_location": x_value,
+    }
+
+
+@router.get(
+    "/compute/univariate_misfits",
+    response_model=Mapping[int, Mapping[str, Any]],
```

Comment:
Can you make this into a `pydantic` model? It's currently difficult to reason about the structure of the returned document, and it's not going to have any structure in Swagger documentation.

I also don't see why this would return a JSON object, rather than a CSV.

Reply:
Ok, convinced this will become CSV.